### PR TITLE
Access gprc services in the gateway in a better way.

### PR DESCRIPTION
**Pull request by midnight2104**
// Describe your PR here; eg. Fixes #issueNo
The shenyu-client-grpc module has already provided the json grpc service that wrap the origin grpc service.
If the request message and response message of the json service can be dynamically constructed in the shenyu-plugin-grpc module, we can call the server.	
Make sure that:

- [ ] You have read the [contribution guidelines](https://dromara.org/projects/soul/contributor/).
- [ ] You submit test cases (unit or integration tests) that back your changes.
- [ ] Your local test passed `mvn clean install -Dmaven.javadoc.skip=true`.

### CODE REVIEW

File: shenyu-client/shenyu-client-grpc/src/main/java/org/apache/shenyu/client/grpc/json/JsonForwardingServerCall.java [hunk 1]
```diff
@@ -52,20 +51,18 @@ public JsonForwardingServerCall(final ServerCall<P, P> call) {
...
 
-            jsonFormat = JsonFormat.printer().includingDefaultValueFields().preservingProtoFieldNames()
+            String jsonFormat = JsonFormat.printer().includingDefaultValueFields().preservingProtoFieldNames()
                     .print((MessageOrBuilder) message);
 
-            rep = JsonReply.newBuilder().setMessage(jsonFormat).build();
-            log.debug("begin send json response: {}", jsonFormat);
-            delegate().sendMessage((P) rep);
+            DynamicMessage respMessage = JsonMessage.buildJsonMessage(jsonFormat);
+            log.info("begin send json response");
```
Comment (dengliming) [inline shenyu-client/shenyu-client-grpc/src/main/java/org/apache/shenyu/client/grpc/json/JsonForwardingServerCall.java:64]: Is there any special meaning here?

Comment (midnight2104) [reply shenyu-client/shenyu-client-grpc/src/main/java/org/apache/shenyu/client/grpc/json/JsonForwardingServerCall.java:64]: emmm, I wanto print some log to see if the response data is sent.

Comment (dengliming) [reply shenyu-client/shenyu-client-grpc/src/main/java/org/apache/shenyu/client/grpc/json/JsonForwardingServerCall.java:64]: I suggest you can use `debug`.

Comment (midnight2104) [reply shenyu-client/shenyu-client-grpc/src/main/java/org/apache/shenyu/client/grpc/json/JsonForwardingServerCall.java:64]: ok~

**Review by SaberSola — APPROVED** [review]

**Pull request merged**